src/python/engagement_edge/tests/test_engagement_edge.py

```diff
@@ -8,19 +8,9 @@
 # gross hack because engagement edge is pseudo singleton
 JWT_SECRET.secret = "hey im a fake secret"
 
-
-class TestEngagementEdgeChalice(unittest.TestCase):
-    def test_requires_auth_fails_without_cookie_headers(self):
```

Comment:
This seems like an important test, maybe we should create a bugfix ticket to refactor this code s.t. we can inject the `BUCKET_PREFIX` data dependency at test time? Regressing on this seems like it could be a security issue, but I'm not terribly familiar with this service...